src/state/slices/opportunitiesSlice/resolvers/thorchainLending/types.ts

```diff
@@ -0,0 +1,95 @@
+export type LendingDepositQuoteResponseSuccess = {
+  inbound_address: string
+  inbound_confirmation_blocks: number
+  inbound_confirmation_seconds: number
+  outbound_delay_blocks: number
+  outbound_delay_seconds: number
+  fees: {
+    asset: string
+    affiliate: string
+    outbound: string
```

Comment:
according to openapi spec
```suggestion
    affiliate?: string
    outbound?: string
```

Reply:
a1a46d9d9ceeee40b39891bf1bca4b25d0c52653